manual/detailedTopics/build/AggregatingReverseRouters.md

```diff
@@ -1,14 +1,32 @@
 <!--- Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com> -->
+<!--
 # Aggregating reverse routers
+-->
+# リバースルートの統合
 
+<!--
 In some situations you want to share reverse routers between sub projects that are not dependent on each other.
+-->
+いくつかの状況において、互いに依存していないサブプロジェクト間でリバースルートを共有したくなることがあります。
 
+<!--
 For example, you might have a `web` sub project, and an `api` sub project.  These sub projects may have no dependence on each other, except that the `web` project wants to render links to the `api` project (for making AJAX calls), while the `api` project wants to render links to the `web` (rendering the web link for a resource in JSON).  In this situation, it would be convenient to use the reverse router, but since these projects don't depend on each other, you can't.
+-->
+例えば、`web` サブプロジェクトと `api` サブプロジェクト があるとします。`web` プロジェクトは `api` プロジェクトへのリンクを (AJAX 呼び出しのために) レンダリングする一方で、`api` プロジェクトは `web` へのリンク (JSON 内のリソースに対する web リンク) をレンダリングすることを除いては、これらのサブプロジェクトはお互いに依存関係を持ちません。このような場合、リバースルータを使うと便利ですが、これらのプロジェクトはお互いに依存していないため、使えません。
 
+<!--
 Play's routes compiler offers a feature that allows a common dependency to generate the reverse routers for projects that depend on it so that the reverse routers can be shared between those projects.  This is configured using the `aggregateReverseRoutes` sbt configuration item, like this:
+-->
+Play のルートコンパイラには、依存するプロジェクト用にリバースルートを生成する共通の依存性を提供する機能があるので、これらのプロジェクト間でリバースルートを共有することができます。以下のように、`aggregateReverseRoutes` sbt 設定項目を使って設定することができます:
```

Comment:
`that depend on it` の it は単数なので、 `Play's routes compiler`, `a feature`, `a common dependency` のいずれかを指すのですが、`depend on` ということなので一番近い `a common dependency` を指すと捉えるのが順当かなと思います

so that は 「〜 して … する」みたいな訳し方ができる場合があるので、

Play のルートコンパイラには、共通の依存性がそれに依存するプロジェクト用にリバースルータを生成することを許可して、これらのプロジェクト間でリバースルートを共有できるようにする機能があります。

などというのはいかがでしょう〜